environments/helm-values/values-int.yaml

```
@@ -30,13 +30,27 @@ bpdm:
   portalGateApiPath: "/companies/test-company/v6"
 custodianAddress: "https://managed-identity-wallets-new.int.catena-x.net"
 sdfactoryAddress: "https://sdfactory.int.catena-x.net"
-clearinghouseAddress: "https://digitalidcore.test.dih-cloud.com"
-clearinghouseTokenAddress: "https://iam.test.dih-cloud.com/realms/development/protocol/openid-connect/token"
 issuerComponentAddress: "https://ssi-credential-issuer.int.catena-x.net"
 bpnDidResolverAddress: "http://bdrs-bdrs-server:8081"
 dimWrapper:
   baseAddress: "https://dim.int.catena-x.net"
   tokenAddress: "https://centralidp.int.catena-x.net/auth/realms/CX-Central/protocol/openid-connect/token"
+clearinghouse:
+  default:
+    baseAddress: "https://digitalidcore.test.dih-cloud.com"
+    apiPath: "/api/v2/validation"
+    tokenAddress: "https://iam.test.dih-cloud.com/realms/development/protocol/openid-connect/token"
+    clientId: "<path:portal/data/int/clearinghouse#client-id>"
+    clientSecret: "<path:portal/data/int/clearinghouse#client-secret>"
+  regional:
+    - countryAlpha2Code: "FR"
+      baseAddress: "https://digitalidcore.test.dih-cloud.com"
+      apiPath: "/api/v2/validation"
+      clientId: "<path:portal/data/int/clearinghouse#client-id-0>"
+      clientSecret: "<path:portal/data/int/clearinghouse#client-secret-0>"
```

I think the secret storage could be a bit more intuitive
```suggestion
      clientId: "<path:portal/data/int/clearinghouse/region/fr#client-id-0>"
      clientSecret: "<path:portal/data/int/clearinghouse/region/fr#client-secret-0>"
```

---

Hi @evegufy, I also had to move default secrets to portal/data/int/clearinghouse/default to make it work. Not sure what to do with stable environment.

---

Hi @evegufy,
the moment I added the path for portal/data/int/clearinghouse/region/fr it removed the secrets at portal/data/int/clearinghouse#. I wanted to test which path works for our setup. I can revert vault changes until this PR is merged if needed.

Helm-docs changes are also now added.